compiler/rustc_codegen_ssa/src/sir.rs

```diff
@@ -215,6 +227,20 @@ impl SirFuncCx<'tcx> {
         this
     }
 
+    /// Compute layout and offsets required for blackholing.
+    pub fn compute_layout_and_offsets(&mut self) {
+        let mut layout = Layout::from_size_align(0, 1).unwrap();
+        for ld in &self.func.local_decls {
+            let (size, align) = self.layout_map.get(&ld.ty).unwrap();
+            let l = Layout::from_size_align(*size, *align).unwrap();
```

Comment:
I wonder if `decl_layout` would be more descriptive, since we have many layouts in scope here.

Reply:
Since we immediately use this variable in the next line and then never again, I'm okay with it being short. But I could merge those two lines into one if you like.

Reply:
Only if you see fit.